google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebuggerClient.java

```diff
@@ -46,8 +46,8 @@
  */
 public class CloudDebuggerClient {
 
-  private static final int LONG_CONNECTION_TIMEOUT_MS = 120 * 1000;
-  private static final int SHORT_CONNECTION_TIMEOUT_MS = 10 * 1000;
+  static final int LONG_CONNECTION_TIMEOUT_MS = 120 * 1000;
```

Comment:
Do you mean used by a subclass or package member?  I don't see any usages
outside of this class.

On Fri, Feb 5, 2016 at 10:48 AM, akerekes <notifications@github.com> wrote:

> In
> google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebuggerClient.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/413#discussion_r52031812>
> :
>
> > @@ -46,8 +46,8 @@
> >   */
> >  public class CloudDebuggerClient {
> >
> > -  private static final int LONG_CONNECTION_TIMEOUT_MS = 120 * 1000;
> > -  private static final int SHORT_CONNECTION_TIMEOUT_MS = 10 * 1000;
> > +  static final int LONG_CONNECTION_TIMEOUT_MS = 120 * 1000;
>
> In this case I think these may be used in other parts of the code later,
> so that's why I have not added the annotation. I actually think these could
> become configurable eventually by the user.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/413/files#r52031812>
> .
>